app/views/articles/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="l-constrained">
+  <div class="hero">
+    <h1 class="hero__heading">
+      HERO component
```

Comment:
I'm assuming this text is going to change in the future?

Reply:
Yes! It will change.